EpsilonAuxBMS/Src/SetContactorsTask.c

```diff
@@ -66,6 +65,8 @@ void setContactorsTask(void const* arg)
     uint8_t common;
     uint8_t charge;
     uint8_t discharge;
+    uint8_t isChargeOn = 0;
```

Comment:
`isChargeOn` implies this is a constant check, when really we only check if it has connected at any point. How about `hasChargeConnected` ?